internal/auth0/branding_mock.go

```diff
@@ -0,0 +1,129 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
do we need this mock?

Reply:
Removed in https://github.com/auth0/auth0-cli/pull/316/commits/46cb25c7903ba70db43d8a58f7fb24bdebb45a89.